bigframes/session/__init__.py

```diff
@@ -338,6 +342,24 @@ def objects(
     def _project(self):
         return self.bqclient.project
 
+    @property
+    def bytes_processed_sum(self):
+        """The sum of all bytes processed by bigquery jobs using this session."""
+        return self._bytes_processed_sum
```

Comment:
Thank you! This benchmark data is valuable for evaluating performance. I'd like to suggest adding the following metrics to enhance our analysis someday (don't have to address to this PR):
1. Number of query jobs
2. Query text length: this will provide insights into the relationship between query complexity (length)
3. Average response times per query or total queries.

Reply:
Great suggestions, thanks. I think those could all be pretty simple to add. Will look into adding them once we get the round trip down for these performance reports. (We're thinking about adding a dashboard.)